src/applications/search/styles.scss

```diff
@@ -128,18 +145,54 @@
 
   .search-box {
 
+  @media (max-width: $small-screen) {
+    flex-direction: column;
+  }
+
     input,
     button {
       margin: 0;
       max-width: none;
+      font-family: "Source Sans Pro";
+
+      @media (max-width: $small-screen) {
```

Comment:
Just a small thought here. I was under the _impression_ that we write styles mobile first. I think that is standard for the VA, but I am not I'm not sure where I herd that or saw it. 
Maybe @kelsonic has more input to whether or not there is a suggested common practice.

Reply:
good to know! This is all legacy code so I was tweaking and adding to existing. happy to rewrite some stuff to make it fit our standards!

Reply:
@zacharymorel that's correct! 🙂 @media queries are usually for min-width: $medium-screen here I've noticed 👍